Project.toml

```diff
@@ -81,4 +79,10 @@ TimerOutputs = "0.5"
 VTKDataTypes = "0.1"
 WriteVTK = "1.9"
 Zygote = "0.6"
-julia = "1.5, 1.6"
+julia = "1.9"
+
+[weakdeps]
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
+
+[extensions]
+TopOptMakieExt = "Makie"
```

Comment:
@yijiangh these 2 new Project.toml sections are where we declare the extension packages